pyfolio/timeseries.py

```diff
@@ -593,11 +593,12 @@ def rolling_regression(returns, factor_returns=None,
     for beg, end in zip(ret_no_na.index[:-rolling_window],
                         ret_no_na.index[rolling_window:]):
         returns_period = ret_no_na[beg:end]
-        reg = linear_model.LinearRegression(fit_intercept=True).fit(
-            factor_returns.loc[returns_period.index],
-            returns_period)
-        rolling_risk.loc[end, factor_returns.columns] = reg.coef_
-        rolling_risk.loc[end, 'alpha'] = reg.intercept_
+        if ~np.any(factor_returns.loc[returns_period.index].isnull()):
```

Comment:
Can't we just drop individual nans rather then dropping the whole period? Also prefer `not` over `~`.

Reply:
@twiecki What if there are a large proportion of NaNs? Then we might only do the regression on 2 days of data instead of 120 days. And sure, I'll switch to not.

Reply:
Right, we should add a test that e.g. not more than 10% of values are missing.